easybuild/easyconfigs/s/ScaLAPACK/ScaLAPACK-2.0.2-gompi-2016.07-OpenBLAS-0.2.18-LAPACK-3.6.1.eb

```diff
@@ -0,0 +1,25 @@
+name = 'ScaLAPACK'
+version = '2.0.2'
+
+homepage = 'http://www.netlib.org/scalapack/'
+description = """The ScaLAPACK (or Scalable LAPACK) library includes a subset of LAPACK routines
+ redesigned for distributed memory MIMD parallel computers."""
+
+toolchain = {'name': 'gompi', 'version': '2016.07'}
+toolchainopts = {'pic': True}
+
+source_urls = [homepage]
+sources = [SOURCELOWER_TGZ]
+
+blaslib = 'OpenBLAS'
+blasver = '0.2.18'
+blassuff = '-LAPACK-3.6.1'
+
+versionsuffix = "-%s-%s%s" % (blaslib, blasver, blassuff)
+
+dependencies = [(blaslib, blasver, blassuff, ('gompi', '2016.07'))]
```

Comment:
@AFCJamie the error in the tests is because you're hardcoding the toolchain to use for this dep to the same toolchain used in this easyconfig, so please change this to:

```python
dependencies = [(blaslib, blasver, blassuff)]
```